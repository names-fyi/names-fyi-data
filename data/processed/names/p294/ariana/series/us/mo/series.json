[{"y":1989,"t":8,"m":0,"f":8,"mr":null,"fr":508,"mp":0,"fp":0.0002459268367660621},{"y":1990,"t":9,"m":0,"f":9,"mr":null,"fr":469,"mp":0,"fp":0.0002739976253539136},{"y":1991,"t":10,"m":0,"f":10,"mr":null,"fr":445,"mp":0,"fp":0.00030792917628945344},{"y":1992,"t":12,"m":0,"f":12,"mr":null,"fr":366,"mp":0,"fp":0.00038806066681757915},{"y":1993,"t":9,"m":0,"f":9,"mr":null,"fr":461,"mp":0,"fp":0.00029481132075471697},{"y":1994,"t":24,"m":0,"f":24,"mr":null,"fr":211,"mp":0,"fp":0.000808979674385681},{"y":1995,"t":12,"m":0,"f":12,"mr":null,"fr":388,"mp":0,"fp":0.000405569825604975},{"y":1996,"t":19,"m":0,"f":19,"mr":null,"fr":258,"mp":0,"fp":0.0006429997631053504},{"y":1997,"t":16,"m":0,"f":16,"mr":null,"fr":314,"mp":0,"fp":0.0005420788724759452},{"y":1998,"t":21,"m":0,"f":21,"mr":null,"fr":265,"mp":0,"fp":0.0007074518258994745},{"y":1999,"t":27,"m":0,"f":27,"mr":null,"fr":220,"mp":0,"fp":0.0009226983801517326},{"y":2000,"t":26,"m":0,"f":26,"mr":null,"fr":235,"mp":0,"fp":0.0008836023789294817},{"y":2001,"t":44,"m":0,"f":44,"mr":null,"fr":139,"mp":0,"fp":0.0015371716042481833},{"y":2002,"t":36,"m":0,"f":36,"mr":null,"fr":168,"mp":0,"fp":0.0012899064817800709},{"y":2003,"t":22,"m":0,"f":22,"mr":null,"fr":262,"mp":0,"fp":0.0007730962504831851},{"y":2004,"t":43,"m":0,"f":43,"mr":null,"fr":142,"mp":0,"fp":0.001516113109089627},{"y":2005,"t":44,"m":0,"f":44,"mr":null,"fr":142,"mp":0,"fp":0.001517398351553609},{"y":2006,"t":47,"m":0,"f":47,"mr":null,"fr":145,"mp":0,"fp":0.0015837180307982613},{"y":2007,"t":46,"m":0,"f":46,"mr":null,"fr":148,"mp":0,"fp":0.0015530047265361242},{"y":2008,"t":42,"m":0,"f":42,"mr":null,"fr":161,"mp":0,"fp":0.0014283770915521697},{"y":2009,"t":32,"m":0,"f":32,"mr":null,"fr":199,"mp":0,"fp":0.0011436740528949249},{"y":2010,"t":40,"m":0,"f":40,"mr":null,"fr":156,"mp":0,"fp":0.0014648258688248433},{"y":2011,"t":34,"m":0,"f":34,"mr":null,"fr":181,"mp":0,"fp":0.001258047805816621},{"y":2012,"t":46,"m":0,"f":46,"mr":null,"fr":135,"mp":0,"fp":0.0017109276203228446},{"y":2013,"t":54,"m":0,"f":54,"mr":null,"fr":118,"mp":0,"fp":0.002001037575038909},{"y":2014,"t":60,"m":0,"f":60,"mr":null,"fr":106,"mp":0,"fp":0.0022154936858429955},{"y":2015,"t":46,"m":0,"f":46,"mr":null,"fr":140,"mp":0,"fp":0.0016937295187598954},{"y":2016,"t":54,"m":0,"f":54,"mr":null,"fr":111,"mp":0,"fp":0.0019938706937931545},{"y":2017,"t":45,"m":0,"f":45,"mr":null,"fr":130,"mp":0,"fp":0.0017353077279037483},{"y":2018,"t":41,"m":0,"f":41,"mr":null,"fr":151,"mp":0,"fp":0.001551795919912191},{"y":2019,"t":26,"m":0,"f":26,"mr":null,"fr":245,"mp":0,"fp":0.00101793125048939},{"y":2020,"t":36,"m":0,"f":36,"mr":null,"fr":162,"mp":0,"fp":0.0014768624876928126},{"y":2021,"t":34,"m":0,"f":34,"mr":null,"fr":179,"mp":0,"fp":0.001380654592706895},{"y":2022,"t":26,"m":0,"f":26,"mr":null,"fr":235,"mp":0,"fp":0.0010839656466271992}]