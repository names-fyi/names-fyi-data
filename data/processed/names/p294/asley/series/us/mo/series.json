[{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":641,"mp":0,"fp":0.00015922045664426966}]