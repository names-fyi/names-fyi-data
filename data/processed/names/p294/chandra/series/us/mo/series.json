[{"y":1958,"t":7,"m":0,"f":7,"mr":null,"fr":521,"mp":0,"fp":0.00015721151686654987},{"y":1959,"t":7,"m":0,"f":7,"mr":null,"fr":524,"mp":0,"fp":0.00015565586711436258},{"y":1963,"t":6,"m":0,"f":6,"mr":null,"fr":572,"mp":0,"fp":0.0001460955952178042},{"y":1964,"t":6,"m":0,"f":6,"mr":null,"fr":580,"mp":0,"fp":0.00015184107301024927},{"y":1966,"t":7,"m":0,"f":7,"mr":null,"fr":483,"mp":0,"fp":0.00020247013565499088},{"y":1967,"t":10,"m":0,"f":10,"mr":null,"fr":399,"mp":0,"fp":0.00029840057292910004},{"y":1968,"t":13,"m":0,"f":13,"mr":null,"fr":341,"mp":0,"fp":0.00039530499300614245},{"y":1969,"t":11,"m":0,"f":11,"mr":null,"fr":383,"mp":0,"fp":0.00032076517073455224},{"y":1970,"t":16,"m":0,"f":16,"mr":null,"fr":328,"mp":0,"fp":0.00045118718628390955},{"y":1971,"t":15,"m":0,"f":15,"mr":null,"fr":335,"mp":0,"fp":0.0004386734514827163},{"y":1972,"t":19,"m":0,"f":19,"mr":null,"fr":283,"mp":0,"fp":0.0006074168797953964},{"y":1973,"t":11,"m":0,"f":11,"mr":null,"fr":362,"mp":0,"fp":0.00037731965835420024},{"y":1974,"t":22,"m":0,"f":22,"mr":null,"fr":242,"mp":0,"fp":0.0007415147123259969},{"y":1975,"t":17,"m":0,"f":17,"mr":null,"fr":278,"mp":0,"fp":0.0005836108345634934},{"y":1976,"t":14,"m":0,"f":14,"mr":null,"fr":319,"mp":0,"fp":0.000474479766827086},{"y":1977,"t":16,"m":0,"f":16,"mr":null,"fr":309,"mp":0,"fp":0.0005164288941966304},{"y":1978,"t":12,"m":0,"f":12,"mr":null,"fr":351,"mp":0,"fp":0.000391491582930967},{"y":1979,"t":8,"m":0,"f":8,"mr":null,"fr":484,"mp":0,"fp":0.0002477930927675391},{"y":1980,"t":7,"m":0,"f":7,"mr":null,"fr":539,"mp":0,"fp":0.0002094930268749626},{"y":1981,"t":11,"m":0,"f":11,"mr":null,"fr":386,"mp":0,"fp":0.0003428927680798005},{"y":1982,"t":15,"m":0,"f":15,"mr":null,"fr":313,"mp":0,"fp":0.00046602665672476466},{"y":1983,"t":20,"m":0,"f":20,"mr":null,"fr":238,"mp":0,"fp":0.0006265271599523839},{"y":1984,"t":6,"m":0,"f":6,"mr":null,"fr":561,"mp":0,"fp":0.00019136314345856987},{"y":1985,"t":7,"m":0,"f":7,"mr":null,"fr":525,"mp":0,"fp":0.00021746559383640374},{"y":1986,"t":15,"m":0,"f":15,"mr":null,"fr":299,"mp":0,"fp":0.000477661369932809},{"y":1987,"t":11,"m":0,"f":11,"mr":null,"fr":378,"mp":0,"fp":0.00035207886566590915},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":677,"mp":0,"fp":0.00015789806101181077},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":625,"mp":0,"fp":0.00018444512757454656},{"y":1990,"t":9,"m":0,"f":9,"mr":null,"fr":476,"mp":0,"fp":0.0002739976253539136},{"y":1991,"t":13,"m":0,"f":13,"mr":null,"fr":364,"mp":0,"fp":0.00040030792917628943},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":732,"mp":0,"fp":0.00016169194450732464},{"y":1994,"t":6,"m":0,"f":6,"mr":null,"fr":639,"mp":0,"fp":0.00020224491859642025},{"y":1995,"t":7,"m":0,"f":7,"mr":null,"fr":593,"mp":0,"fp":0.00023658239826956874},{"y":1996,"t":8,"m":0,"f":8,"mr":null,"fr":539,"mp":0,"fp":0.0002707367423601476},{"y":1998,"t":6,"m":0,"f":6,"mr":null,"fr":709,"mp":0,"fp":0.00020212909311413557}]