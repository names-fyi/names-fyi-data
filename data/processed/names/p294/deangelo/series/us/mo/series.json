[{"y":1979,"t":5,"m":5,"f":0,"mr":489,"fr":null,"mp":0.00013605442176870748,"fp":0},{"y":1980,"t":7,"m":7,"f":0,"mr":393,"fr":null,"mp":0.0001873260543780775,"fp":0},{"y":1983,"t":5,"m":5,"f":0,"mr":466,"fr":null,"mp":0.00013949725190413748,"fp":0},{"y":1984,"t":7,"m":7,"f":0,"mr":383,"fr":null,"mp":0.00019630938358853554,"fp":0},{"y":1985,"t":5,"m":5,"f":0,"mr":494,"fr":null,"mp":0.00013550135501355014,"fp":0},{"y":1986,"t":5,"m":5,"f":0,"mr":509,"fr":null,"mp":0.00013902018573096813,"fp":0},{"y":1987,"t":5,"m":5,"f":0,"mr":497,"fr":null,"mp":0.00013989535827201252,"fp":0},{"y":1988,"t":14,"m":14,"f":0,"mr":278,"fr":null,"mp":0.0003835091083413231,"fp":0},{"y":1989,"t":10,"m":10,"f":0,"mr":357,"fr":null,"mp":0.00026496383243687237,"fp":0},{"y":1990,"t":16,"m":16,"f":0,"mr":271,"fr":null,"mp":0.0004214408007375214,"fp":0},{"y":1991,"t":7,"m":7,"f":0,"mr":471,"fr":null,"mp":0.00018783878065797242,"fp":0},{"y":1992,"t":12,"m":12,"f":0,"mr":339,"fr":null,"mp":0.00033067871806883627,"fp":0},{"y":1993,"t":12,"m":12,"f":0,"mr":341,"fr":null,"mp":0.0003363700070077085,"fp":0},{"y":1994,"t":10,"m":10,"f":0,"mr":377,"fr":null,"mp":0.000288043321715586,"fp":0},{"y":1995,"t":6,"m":6,"f":0,"mr":543,"fr":null,"mp":0.00017497812773403323,"fp":0},{"y":1996,"t":11,"m":11,"f":0,"mr":368,"fr":null,"mp":0.00031885906429358224,"fp":0},{"y":1997,"t":10,"m":10,"f":0,"mr":402,"fr":null,"mp":0.00029068922415046073,"fp":0},{"y":1998,"t":6,"m":6,"f":0,"mr":577,"fr":null,"mp":0.00017326518236160444,"fp":0},{"y":1999,"t":6,"m":6,"f":0,"mr":583,"fr":null,"mp":0.00017575722069248346,"fp":0},{"y":2000,"t":6,"m":6,"f":0,"mr":598,"fr":null,"mp":0.00017458608548898653,"fp":0},{"y":2001,"t":6,"m":6,"f":0,"mr":613,"fr":null,"mp":0.00017952247022919035,"fp":0},{"y":2003,"t":9,"m":9,"f":0,"mr":491,"fr":null,"mp":0.0002669910113026195,"fp":0},{"y":2005,"t":7,"m":7,"f":0,"mr":619,"fr":null,"mp":0.00020585208057638583,"fp":0},{"y":2006,"t":11,"m":11,"f":0,"mr":466,"fr":null,"mp":0.00031486146095717883,"fp":0},{"y":2008,"t":5,"m":5,"f":0,"mr":844,"fr":null,"mp":0.00014499898500710496,"fp":0},{"y":2010,"t":7,"m":7,"f":0,"mr":651,"fr":null,"mp":0.00021450678760763644,"fp":0},{"y":2011,"t":9,"m":9,"f":0,"mr":557,"fr":null,"mp":0.00027820710973724884,"fp":0},{"y":2013,"t":6,"m":6,"f":0,"mr":721,"fr":null,"mp":0.00018919089361165415,"fp":0},{"y":2019,"t":5,"m":5,"f":0,"mr":881,"fr":null,"mp":0.00016844091092844631,"fp":0},{"y":2020,"t":5,"m":5,"f":0,"mr":859,"fr":null,"mp":0.00017608113818847726,"fp":0}]