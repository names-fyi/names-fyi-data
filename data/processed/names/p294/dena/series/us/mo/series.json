[{"y":1915,"t":5,"m":0,"f":5,"mr":null,"fr":491,"mp":0,"fp":0.00018924340486734038},{"y":1917,"t":6,"m":0,"f":6,"mr":null,"fr":454,"mp":0,"fp":0.00021003255504603214},{"y":1918,"t":5,"m":0,"f":5,"mr":null,"fr":523,"mp":0,"fp":0.00016268098259313488},{"y":1927,"t":6,"m":0,"f":6,"mr":null,"fr":459,"mp":0,"fp":0.0001988400994200497},{"y":1937,"t":6,"m":0,"f":6,"mr":null,"fr":405,"mp":0,"fp":0.0002313386798272671},{"y":1940,"t":5,"m":0,"f":5,"mr":null,"fr":443,"mp":0,"fp":0.00017880131597768558},{"y":1943,"t":6,"m":0,"f":6,"mr":null,"fr":435,"mp":0,"fp":0.00018235973497051852},{"y":1945,"t":5,"m":0,"f":5,"mr":null,"fr":451,"mp":0,"fp":0.0001672688344707614},{"y":1946,"t":5,"m":0,"f":5,"mr":null,"fr":515,"mp":0,"fp":0.000136317784018103},{"y":1947,"t":13,"m":0,"f":13,"mr":null,"fr":303,"mp":0,"fp":0.00031674097897327195},{"y":1949,"t":6,"m":0,"f":6,"mr":null,"fr":478,"mp":0,"fp":0.0001548586914440573},{"y":1951,"t":9,"m":0,"f":9,"mr":null,"fr":389,"mp":0,"fp":0.0002187332912069217},{"y":1952,"t":5,"m":0,"f":5,"mr":null,"fr":551,"mp":0,"fp":0.00012098335269066976},{"y":1953,"t":10,"m":0,"f":10,"mr":null,"fr":395,"mp":0,"fp":0.00023860084464699004},{"y":1954,"t":15,"m":0,"f":15,"mr":null,"fr":308,"mp":0,"fp":0.00034551862345380416},{"y":1955,"t":22,"m":0,"f":22,"mr":null,"fr":243,"mp":0,"fp":0.000505514705882353},{"y":1956,"t":18,"m":0,"f":18,"mr":null,"fr":285,"mp":0,"fp":0.00040186644638431825},{"y":1957,"t":18,"m":0,"f":18,"mr":null,"fr":293,"mp":0,"fp":0.00039840637450199205},{"y":1958,"t":18,"m":0,"f":18,"mr":null,"fr":296,"mp":0,"fp":0.00040425818622827114},{"y":1959,"t":31,"m":0,"f":31,"mr":null,"fr":222,"mp":0,"fp":0.0006893331257921772},{"y":1960,"t":35,"m":0,"f":35,"mr":null,"fr":210,"mp":0,"fp":0.0007820180534453481},{"y":1961,"t":26,"m":0,"f":26,"mr":null,"fr":248,"mp":0,"fp":0.0005870134561546103},{"y":1962,"t":25,"m":0,"f":25,"mr":null,"fr":252,"mp":0,"fp":0.0005868544600938967},{"y":1963,"t":31,"m":0,"f":31,"mr":null,"fr":221,"mp":0,"fp":0.000754827241958655},{"y":1964,"t":32,"m":0,"f":32,"mr":null,"fr":224,"mp":0,"fp":0.0008098190560546628},{"y":1965,"t":14,"m":0,"f":14,"mr":null,"fr":341,"mp":0,"fp":0.00038435140700068636},{"y":1966,"t":33,"m":0,"f":33,"mr":null,"fr":204,"mp":0,"fp":0.0009545020680878142},{"y":1967,"t":29,"m":0,"f":29,"mr":null,"fr":213,"mp":0,"fp":0.0008653616614943901},{"y":1968,"t":26,"m":0,"f":26,"mr":null,"fr":233,"mp":0,"fp":0.0007906099860122849},{"y":1969,"t":38,"m":0,"f":38,"mr":null,"fr":184,"mp":0,"fp":0.0011080978625375442},{"y":1970,"t":51,"m":0,"f":51,"mr":null,"fr":147,"mp":0,"fp":0.0014381591562799617},{"y":1971,"t":41,"m":0,"f":41,"mr":null,"fr":172,"mp":0,"fp":0.001199040767386091},{"y":1972,"t":36,"m":0,"f":36,"mr":null,"fr":181,"mp":0,"fp":0.0011508951406649615},{"y":1973,"t":41,"m":0,"f":41,"mr":null,"fr":146,"mp":0,"fp":0.0014063732720474736},{"y":1974,"t":31,"m":0,"f":31,"mr":null,"fr":185,"mp":0,"fp":0.0010448616400957227},{"y":1975,"t":24,"m":0,"f":24,"mr":null,"fr":230,"mp":0,"fp":0.0008239211782072848},{"y":1976,"t":22,"m":0,"f":22,"mr":null,"fr":238,"mp":0,"fp":0.0007456110621568494},{"y":1977,"t":19,"m":0,"f":19,"mr":null,"fr":264,"mp":0,"fp":0.0006132593118584984},{"y":1978,"t":16,"m":0,"f":16,"mr":null,"fr":303,"mp":0,"fp":0.0005219887772412893},{"y":1979,"t":12,"m":0,"f":12,"mr":null,"fr":363,"mp":0,"fp":0.0003716896391513087},{"y":1980,"t":18,"m":0,"f":18,"mr":null,"fr":286,"mp":0,"fp":0.0005386963548213323},{"y":1981,"t":8,"m":0,"f":8,"mr":null,"fr":487,"mp":0,"fp":0.00024937655860349125},{"y":1982,"t":9,"m":0,"f":9,"mr":null,"fr":434,"mp":0,"fp":0.0002796159940348588},{"y":1983,"t":10,"m":0,"f":10,"mr":null,"fr":409,"mp":0,"fp":0.00031326357997619195},{"y":1984,"t":13,"m":0,"f":13,"mr":null,"fr":332,"mp":0,"fp":0.00041462014416023473},{"y":1985,"t":6,"m":0,"f":6,"mr":null,"fr":582,"mp":0,"fp":0.00018639908043120322},{"y":1986,"t":9,"m":0,"f":9,"mr":null,"fr":453,"mp":0,"fp":0.0002865968219596854},{"y":1987,"t":10,"m":0,"f":10,"mr":null,"fr":414,"mp":0,"fp":0.00032007169605991745},{"y":1988,"t":11,"m":0,"f":11,"mr":null,"fr":387,"mp":0,"fp":0.0003473757342259837},{"y":1989,"t":7,"m":0,"f":7,"mr":null,"fr":573,"mp":0,"fp":0.00021518598217030432},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":645,"mp":0,"fp":0.00018475750577367207},{"y":1997,"t":5,"m":0,"f":5,"mr":null,"fr":799,"mp":0,"fp":0.00016939964764873288}]