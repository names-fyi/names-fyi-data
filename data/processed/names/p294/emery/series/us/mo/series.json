[{"y":1910,"t":5,"m":5,"f":0,"mr":165,"fr":null,"mp":0.001004016064257028,"fp":0},{"y":1911,"t":7,"m":7,"f":0,"mr":137,"fr":null,"mp":0.0013155421913174216,"fp":0},{"y":1912,"t":5,"m":5,"f":0,"mr":267,"fr":null,"mp":0.0004474673348845534,"fp":0},{"y":1913,"t":16,"m":16,"f":0,"mr":141,"fr":null,"mp":0.0011465424579003942,"fp":0},{"y":1914,"t":8,"m":8,"f":0,"mr":263,"fr":null,"mp":0.0004678636177554243,"fp":0},{"y":1915,"t":10,"m":10,"f":0,"mr":279,"fr":null,"mp":0.0004218341348181895,"fp":0},{"y":1916,"t":8,"m":8,"f":0,"mr":322,"fr":null,"mp":0.0003251239535072746,"fp":0},{"y":1917,"t":8,"m":8,"f":0,"mr":328,"fr":null,"mp":0.0003270779672104338,"fp":0},{"y":1918,"t":8,"m":8,"f":0,"mr":338,"fr":null,"mp":0.00029890898221491554,"fp":0},{"y":1919,"t":10,"m":10,"f":0,"mr":288,"fr":null,"mp":0.00039436841897700834,"fp":0},{"y":1920,"t":13,"m":13,"f":0,"mr":255,"fr":null,"mp":0.00046670256686411776,"fp":0},{"y":1921,"t":13,"m":13,"f":0,"mr":256,"fr":null,"mp":0.00044288488399822846,"fp":0},{"y":1922,"t":11,"m":11,"f":0,"mr":281,"fr":null,"mp":0.0003905140585061062,"fp":0},{"y":1923,"t":6,"m":6,"f":0,"mr":392,"fr":null,"mp":0.00020554280428899317,"fp":0},{"y":1924,"t":13,"m":13,"f":0,"mr":257,"fr":null,"mp":0.00042694341357680057,"fp":0},{"y":1925,"t":10,"m":10,"f":0,"mr":279,"fr":null,"mp":0.0003372340066772333,"fp":0},{"y":1926,"t":7,"m":7,"f":0,"mr":353,"fr":null,"mp":0.00023734445461634965,"fp":0},{"y":1927,"t":12,"m":12,"f":0,"mr":254,"fr":null,"mp":0.00041239947762732835,"fp":0},{"y":1928,"t":6,"m":6,"f":0,"mr":374,"fr":null,"mp":0.00021521575379317766,"fp":0},{"y":1929,"t":6,"m":6,"f":0,"mr":345,"fr":null,"mp":0.00022066124820712736,"fp":0},{"y":1930,"t":7,"m":7,"f":0,"mr":326,"fr":null,"mp":0.0002484736617918501,"fp":0},{"y":1932,"t":7,"m":7,"f":0,"mr":312,"fr":null,"mp":0.0002532561505065123,"fp":0},{"y":1933,"t":7,"m":7,"f":0,"mr":319,"fr":null,"mp":0.00026195644038619863,"fp":0},{"y":1934,"t":7,"m":7,"f":0,"mr":326,"fr":null,"mp":0.00025484199796126404,"fp":0},{"y":1935,"t":7,"m":7,"f":0,"mr":307,"fr":null,"mp":0.00026116479498563595,"fp":0},{"y":1940,"t":5,"m":5,"f":0,"mr":384,"fr":null,"mp":0.00017123874105277577,"fp":0},{"y":1941,"t":7,"m":7,"f":0,"mr":328,"fr":null,"mp":0.00022572635516429653,"fp":0},{"y":1947,"t":5,"m":5,"f":0,"mr":432,"fr":null,"mp":0.00011441647597254005,"fp":0},{"y":1950,"t":5,"m":5,"f":0,"mr":408,"fr":null,"mp":0.0001199616122840691,"fp":0},{"y":2004,"t":5,"m":0,"f":5,"mr":null,"fr":892,"mp":0,"fp":0.00017629222198716594},{"y":2005,"t":8,"m":0,"f":8,"mr":null,"fr":647,"mp":0,"fp":0.00027589060937338345},{"y":2006,"t":12,"m":0,"f":12,"mr":null,"fr":485,"mp":0,"fp":0.00040435353977827946},{"y":2007,"t":14,"m":0,"f":14,"mr":null,"fr":435,"mp":0,"fp":0.0004726536124240378},{"y":2008,"t":34,"m":6,"f":28,"mr":752,"fr":239,"mp":0.00017399878200852594,"fp":0.0009522513943681131},{"y":2009,"t":39,"m":10,"f":29,"mr":522,"fr":215,"mp":0.00029910567403463646,"fp":0.0010364546104360258},{"y":2010,"t":31,"m":0,"f":31,"mr":null,"fr":199,"mp":0,"fp":0.0011352400483392536},{"y":2011,"t":45,"m":15,"f":30,"mr":386,"fr":211,"mp":0.0004636785162287481,"fp":0.001110042181602901},{"y":2012,"t":59,"m":9,"f":50,"mr":555,"fr":127,"mp":0.00028367006020109056,"fp":0.0018597039351335267},{"y":2013,"t":50,"m":9,"f":41,"mr":565,"fr":150,"mp":0.0002837863404174812,"fp":0.0015193063069739866},{"y":2014,"t":55,"m":5,"f":50,"mr":891,"fr":122,"mp":0.00015475564084310874,"fp":0.001846244738202496},{"y":2015,"t":75,"m":10,"f":65,"mr":531,"fr":98,"mp":0.00031686682087518614,"fp":0.0023933134504215913},{"y":2016,"t":82,"m":8,"f":74,"mr":646,"fr":77,"mp":0.0002535496957403651,"fp":0.002732341321123952},{"y":2017,"t":79,"m":7,"f":72,"mr":712,"fr":77,"mp":0.00022923011428758555,"fp":0.002776492364645997},{"y":2018,"t":80,"m":7,"f":73,"mr":697,"fr":74,"mp":0.0002312367864693446,"fp":0.0027629537110631696},{"y":2019,"t":94,"m":11,"f":83,"mr":522,"fr":58,"mp":0.0003705700040425819,"fp":0.0032495497611776682},{"y":2020,"t":91,"m":10,"f":81,"mr":547,"fr":54,"mp":0.0003521622763769545,"fp":0.003322940597308828},{"y":2021,"t":77,"m":0,"f":77,"mr":null,"fr":63,"mp":0,"fp":0.0031267765776009098},{"y":2022,"t":91,"m":8,"f":83,"mr":654,"fr":50,"mp":0.0002885378345235519,"fp":0.0034603518719252897}]