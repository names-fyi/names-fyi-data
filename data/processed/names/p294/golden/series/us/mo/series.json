[{"y":1910,"t":5,"m":0,"f":5,"mr":null,"fr":313,"mp":0,"fp":0.00045749839875560436},{"y":1916,"t":5,"m":0,"f":5,"mr":null,"fr":517,"mp":0,"fp":0.0001733402669440111},{"y":1919,"t":7,"m":7,"f":0,"mr":356,"fr":null,"mp":0.00027605789328390585,"fp":0},{"y":1920,"t":5,"m":5,"f":0,"mr":441,"fr":null,"mp":0.0001795009872554299,"fp":0}]